tenant/pkg/controller/tenant/tenant_controller.go

```diff
@@ -65,6 +65,14 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
+
+	// Watch for changes to namespaces
+	err = c.Watch(&source.Kind{Type: &corev1.Namespace{}}, &handler.EnqueueRequestForOwner{
```

Comment:
Does it make sense to scope this via labels/annotations so we don't have to process changes to namespaces not used by the tenant?

Reply:
“EnqueueRequestForOwner” makes sure only the namespace that has a tenant as ownership will be put into reconcile request if it has CUD operations.

Reply:
Thanks for the patience. You can tell I've spent most of my time in other controller frameworks.